### ISSUE TITLE
Parsing error loaning custom advancement

### ISSUE THREAD

**Issue by MobButcher** [ConsistencyPlus/ConsistencyPlus#52]
**Describe the bug**
While loading a world, an error appears in console. I thought I would give you a bug report so you would check it out.

**Versions**
Minecraft version: 1.17.1
C+ version: 0.4.0
Fabric API version: 0.11.6

**Other mods**
List all mods that you're using; could be in the form of a list of the filenames, a crash report list, a screenshot of
the mods folder, etc. Just make sure all mods used are known!
Mods:
```
  [✔️] advancements-enlarger-0.2.4
  [✔️] alternate-current-mc1.17-0.2.0
  [✔️] antighost-1.17.1-fabric0.36.1-1.1.4
  [✔️] appleskin-fabric-mc1.17-2.1.2
  [✔️] architectury-2.3.24-fabric
  [✔️] axolotl-item-fix-1.1.1
  [✔️] bclib-0.2.4
  [✔️] betterbeds-1.2.0
  [✔️] betterdroppeditems-1.3.0-1.17
  [✔️] better-end-0.10.5-pre
  [✔️] betterf3-1.1.5+1.17
  [✔️] BetterMineshafts-Fabric-1.17-1.0.0
  [✔️] better-nether-5.1.3
  [✔️] BetterStrongholds-Fabric-1.17-1.1
  [✔️] BetterThirdPerson-Fabric-1.17-1.5.0
  [✔️] blame-1.17-4.0.1-fabric
  [✔️] builtin-servers-1.0.1-BETA+1.17
  [✔️] caelus-fabric-0.0.17-1.17
  [✔️] capes-1.2.0
  [✔️] carpet-extra-1.17.1-1.4.43
  [✔️] charm-fabric-1.17-3.3.2
  [✔️] charmonium-fabric-1.17-3.3.0
  [✔️] ChickenDropFeathers-1.0-fabric-0.36.0+1.17
  [✔️] cleancut-1.17-4.0-fabric
  [✔️] cleardespawn-1.17.1-fabric0.36.1-1.1.1
  [✔️] cloth-api-2.0.54
  [✔️] cloth-config-5.0.38-fabric
  [✔️] consistency_plus-1.17.1-0.4.0
  [✔️] Couplings-1.5.1+1.17
  [✔️] Croptopia-1.17-FABRIC-1.6.0
  [✔️] cullleaves-2.2.0
  [✔️] custom-fog-1.6.0
  [✔️] custom-lan-1.17-v1.0.0
  [✔️] DangerousStoneCutter-1.17-1.5
  [✔️] deathlog-0.1.3
  [✔️] dynamic-fps-2.0.4
  [✔️] dynamic-music-1.3.1
  [✔️] easiervillagertrading-1.17.1-fabric0.36.1-1.5.4
  [✔️] ecotones-0.8.1
  [✔️] elytratrinket-fabric-2.0.1-1.17
  [✔️] EntityCulling-Fabric-1.3.3
  [✔️] fabric-api-0.37.0+1.17
  [✔️] fabric-carpet-1.17.1-1.4.44+v210714
  [✔️] fabric-chunkpregen-0.3.4
  [✔️] fabric-language-kotlin-1.6.3+kotlin.1.5.21
  [✔️] fabrictailor-1.6.1
  [✔️] fabrishot-1.5.1
  [✔️] fallingleaves-1.7.2+1.17
  [✔️] FallingTree-1.17.1-2.12.3
  [✔️] fast-chest-1.2+1.17
  [✔️] FatExperienceOrbs-1.0.1
  [✔️] ferritecore-3.0.1-fabric
  [✔️] FirstPersonMod-2.0.2
  [✔️] get-off-my-lawn-1.17-1.4.0-beta
  [✔️] Harvest-fabric-1.16.1-1.2.12-26
  [✔️] here-be-no-dragons-1.0.0
  [✔️] hydrogen-fabric-mc1.17.1-0.3
  [✔️] identity-1.14.2-beta-1.17
  [✔️] indium-1.0.0+mc1.17.1
  [✔️] item-model-fix-1.0.2+1.17
  [✔️] keybindsgalore-1.0
  [✔️] krypton-0.1.4
  [✔️] lambdynamiclights-2.0.1+1.17
  [✔️] lapisreserve-1.0.8
  [✔️] lithium-fabric-mc1.17.1-0.7.3
  [✔️] mcct-1.4.0-1.17.1
  [✔️] megane-5.1.2
  [✔️] MemoryUsageScreen-1.17-1.1
  [✔️] modmenu-2.0.2
  [✔️] mostructures-1.3.0-pre1-1.17.1-fabric
  [✔️] mousewheelie-1.7.3-newconfig.2+mc1.17.1-pre1
  [✔️] music-duration-reducer-1.2.1
  [✔️] nofade-1.17-2.0.0
  [✔️] notenoughanimations-1.3.0
  [✔️] notenoughcrashes-3.4.0+1.17-fabric
  [✔️] OfflineSkins-1.17.1-v1-fabric
  [✔️] onsoulfire-1.17-3
  [✔️] Patchouli-1.17-54-FABRIC
  [✔️] Paxi-Fabric-1.17-1.2
  [✔️] pling-1.4.0
  [✔️] PresenceFootsteps-r31-1.17-rc1
  [✔️] recipecache-0.2.0-1.17.1
  [✔️] repurposed_structures_fabric-2.1.11+1.17.1
  [✔️] reroll-1.2.1-1.16.5
  [✔️] riverredux-0.2.0
  [✔️] RoughlyEnoughItems-6.0.267-alpha-fabric
  [✔️] seamless-loading-screen-1.3.6+1.17.1
  [✔️] servertick-mc1.17-1.5
  [✔️] slight-gui-modifications-2.0.1
  [✔️] smoothboot-fabric-1.16.5-1.6.0
  [✔️] snow-under-trees-fabric-mc1.17.0-1.0.0
  [✔️] sodium-extra-mc1.17.1-0.3.3
  [✔️] sodium-fabric-mc1.17.1-0.3.0+build.5
  [✔️] tooltipfix-1.0.3-1.17
  [✔️] TRansliterationLib-1.1.0
  [✔️] traverse-4.0.0-beta.2
  [✔️] trinketofundying-fabric-2.0.0-1.17
  [✔️] trinkets-3.0.2
  [✔️] vanillaplusbiomes-0.3.2
  [✔️] villagerfix-2.1.3+1.17
  [✔️] voicechat-fabric-1.17.1-1.1.5
  [✔️] wanderingcollector-1.0.2+mc1.17
  [✔️] WI-Zoom-1.3-MC1.17.1
  [✔️] wthit-fabric-3.8.1
  [✔️] yosbr-0.1.1
  [✔️] YungsApi-1.17-Fabric-14
```

**Logs**
*Even if the game does not crash they may be useful!*
```[16:55:32] [Server thread/ERROR]: Parsing error loading custom advancement consistency_plus:recipes/andesite/andesite_brick_gate_crafting: Expected advancement to be a JsonObject, was "con...te"```

Full report:
https://mclo.gs/8fD415K

**To Reproduce**
Steps to reproduce the behavior. Format it as a numbered list of steps.

1. Launch the game.
2. Load into a world.
3. Observe.

**Expected behavior**
No error appears in the console.

**Additional info**
Thank you for the mod!

**Comment by Cart3r1234** [comment]
This one slipped through 0.4, I'm pretty sure we've already fixed it on our end, so it should be resolved when we release 0.4.1 sometime soon

**Comment by Cart3r1234** [comment]
This should be resolved in 0.4.1rc1
